.github/workflows/goreleaser.yaml

```diff
@@ -48,3 +52,12 @@ jobs:
           args: release --clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Sign container image
+        run: |
+          export TAG=(make version)
+          cosign sign --key cosign.key \
+            ghcr.io/${{ github.repository }}/kubectl-retina:$TAG \
+            ghcr.io/${{ github.repository }}/kubectl-retina:$TAG-linux-amd64 \
+            ghcr.io/${{ github.repository }}/kubectl-retina:$TAG-linux-arm64
+        env:
+          COSIGN_PASSWORD: "" 
```

Comment:
I don't think we need cosign in the GoReleaser job at all - it is only used to produce binaries for kubectl-retina. Attaching binaries to the releases supersedes the kubectl-retina image and we don't build or publish it anymore.

Reply:
Okie, will remove this in the next commit

Reply:
Removed :D